lib/web_console/templates/console.js.erb

```diff
@@ -405,6 +418,17 @@ REPLConsole.prototype.install = function(container) {
     container.parentNode.removeChild(container);
   }
 
+   function minimizeContainer(ev) {
+    var isMinimized = container.style.display === 'none'
+    container.style.display = isMinimized ? 'block' : 'none';
+    minimizedConsole.style.display = isMinimized ? 'none' : 'block';
+    _this.setCookie('console_minimized', isMinimized ? 'false' : 'true', 1);
```

Comment:
Can you use `this.commandStorage` (a Local Storage backed store) instead of a cookie?

Reply:
If I am honest, I am not sure without looking into how it works in full. From what I saw, my guess is: maybe with some work to it. Perhaps with a little bit work it could do he job of the cookies. It clearly persists as the console remembers stuff after a page refresh, I just don't know if I can wack anything into it. 

@rafaelfranca  do you have any thoughts on how it works and if it can support this?

Also, If this pr gets accepted, I actually would like to do more with the front end regarding remembering console height. position, text size and perhaps even some simple styling options such as opacity to see behind the console. If command storage can support that then I don't see why not.

Reply:
It does, it's a wrapper for the native LocalStorage browser API. All modern browsers supports it.

To be even more honest, our frontend has been neglected for years, a clean slate rewrite can be the best thing I do. There is no need on the Action View dependency as well.